### PR TITLE
Added example for Flux Led to use effect:random

### PR DESCRIPTION
**Pull request by tchellomello**
Added on the documentation an example to define ```effect: random``` for Flex Led light bulbs

### CODE REVIEW

File: source/_components/light.flux_led.markdown [hunk 1]
```diff
@@ -54,7 +54,7 @@ light:
     automatic_add: True
 ```
 
-Will add two lights with given name:
+Will add two lights with given name and create an automation rule to randomically set color each 45 seconds:
```
Comment (Landrash) [inline source/_components/light.flux_led.markdown]: Shouldn't it be `randomly ` instead of `randomically`?

Comment (tchellomello) [inline source/_components/light.flux_led.markdown]: Yes, make sense. I'll fix it. Thank you

**Comment by tchellomello** [comment]
Fixed!

**Comment by fabaff** [comment]
Thanks

**Pull request merged**